backend/server/db/helpers/refresh_tokens.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+from time import time
+from typing import Optional
+
+import pymongo
+import pymongo.errors
+
+from server.db.mongo.conn import refreshTokensNewCOL
+
+from .models import RefreshToken, RefreshTokenInfoModel, SessionID
+
+def get_refresh_token_info(token: RefreshToken) -> Optional[RefreshTokenInfoModel]:
+    info = refreshTokensNewCOL.find_one({ 'token': token })
+
+    if info is None:
+        return None
+
+    exp = int(info["expiresAt"].timestamp())
+    if exp <= int(time()):
+        return None
+
+    return RefreshTokenInfoModel(
+        sid=SessionID(info["sid"]),
+        expires_at=exp,
+    )
```

Comment:
How do expired tokens get cleaned up?

Reply:
I have put a ttl index on the documents based on the expiresAt field, which will handle it for us which is crazy cool